src/Resources/config/services.yml

```diff
@@ -1,7 +1,7 @@
 services:
     partnermarketing_file_system.factory:
-        class: "Partnermarketing\FileSystemBundle\Factory\FileSystemFactory"
+        class: "Partnermarketing\\FileSystemBundle\\Factory\\FileSystemFactory"
```

Comment:
Support for quoted single backslashes for FQCNs was deprecated in 2.8, removed in 3.0. If the double backslashes are abhorent to you, we can also remove the quotes and keep a single backslash, but anecdotally I've seen this pattern be more popular.